README.md

```diff
@@ -139,44 +145,44 @@ the `LovelyEditor` with a new valid editorState. You can find a similar example
 [in our Storybook](https://lovely-editor.netlify.com/?selectedKind=App%20Example&selectedStory=with%20an%20example%20Menu%20and%20EditorQuill%20Block&full=0&addons=1&stories=1&panelRight=0&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel).
 
 ```js
-import { LovelyEditor } from 'lovely-editor'
-import { EditorQuill } from 'lovely-editor/dist/components/editor-quill'
+import { LovelyEditor } from "lovely-editor";
+import { EditorQuill } from "lovely-editor/dist/components/editor-quill";
```

Comment:
will the next contributor change the formatting back? think this is not our default formatting